src/main/kotlin/pcimcioch/gitlabci/dsl/GitlabCiDsl.kt

```diff
@@ -97,6 +97,12 @@ fun gitlabCi(validate: Boolean = true, writer: Writer? = null, block: GitlabCiDs
     }
 }
 
+fun gitlabCi(validate: Boolean = true, path: String, block: GitlabCiDsl.() -> Unit) {
+    val writer = FileWriter(path)
+    gitlabCi(validate, writer, block)
+    writer.close()
```

Comment:
NIT: to make sure that writer is closed it's better to use `use` method. Similar to what is done in line 94